.github/workflows/build.yml

```diff
@@ -196,4 +194,25 @@ jobs:
           tags: |
             "${{ secrets.GCP_AR_PARABOL_DEV }}:${{github.sha}}"
             "${{ env.DOCKER_REPOSITORY_FOR_REF }}:${{ env.GITHUB_REF_NAME_NORMALIZED }}"
-          # platforms: linux/amd64,linux/arm64 # Uncomment to be able to build multi-platform images.
+      - name: Push Artifacts to Sentry
+        if: github.ref_type == 'tag'
+        uses: getsentry/action-release@v1
+        env:
+          SENTRY_AUTH_TOKEN: "${{secrets.SENTRY_AUTH_TOKEN}}"
+          SENTRY_ORG: "parabol"
+          SENTRY_PROJECT: "action-production"
+        with:
+          environment: "production"
+          sourcemaps: "./build"
+          version: ${{env.ACTION_VERSION}}
+      - name: Push Artifacts to Datadog
+        if: github.ref_type == 'tag'
+        env:
+          DATADOG_API_KEY: "${{secrets.DATADOG_API_KEY}}"
+        run: |
+          echo "$DATADOG_API_KEY"
+          yarn add @datadog/datadog-ci -W
+          yarn datadog-ci sourcemaps upload ./build \
+            --service=parabol-saas-production \
+            --release-version=${{env.ACTION_VERSION}} \
+            --minified-path-prefix=https://action-files.parabol.co/production/build/
```

Comment:
This must be a variable. Once we migrate to GCP GCS the URL changes. Also, we might want to hide it

Reply:
I believe the plan is to change the DNS record for action-files to point to GCS instead of S3, right?

Reply:
That is correct. Still, I feel it better also more secure) to manage that with a variable.